ui/package.json

```diff
@@ -41,7 +41,7 @@
     "react": "^16.14.0",
     "react-app-polyfill": "^2.0.0",
     "react-bootstrap": "^2.0.2",
-    "react-dev-utils": "^11.0.3",
+    "react-dev-utils": "^11.0.4",
```

Comment:
https://github.com/facebook/create-react-app/blob/v4.0.3/packages/react-dev-utils/package.json

Reply:
Issue is that `react-dev-utils` master branch code is pointing to  immer:9.0.6

but latest release is still pointing to "immer": "8.0.1".

So, I don't think this fix will solve the problem. we need to wait for new release.

Reply:
dependabot also saying the same thing.

![image](https://user-images.githubusercontent.com/5889404/142470527-c4fa9ec5-5104-49dd-8c97-73970eb0e740.png)

Reply:
you're right... I check `react-dev-utils` sources, but missed that the `9.0.6` upgrade was not released yet :facepalm:

Reply:
cool. Let's wait for the new release. which dependabot will auto update when happens. we can sit back and relax !

Reply:
however https://github.com/facebook/create-react-app/blob/main/packages/react-dev-utils/package.json shows `immer` at `9.0.6` in `react-dev-utils` `11.0.4` :thinking:

Reply:
Should we try v. `12.0.0-next.47` for `react-dev-utils`?

Reply:
True. Confusing. I checked from last release tag which is mapping to 11.0.3. 
not sure how 11.0.4 is out if version is still in master branch !

Reply:
Are the independently releasing packages without sync ?

Reply:
`11.0.4` shows available here: https://www.npmjs.com/package/react-dev-utils/v/11.0.4

Reply:
but a local build with 11.0.4 still pulls `immer` 8.0.1 :(

Reply:
however, `immer` is not pinned, it seems, let me try to bump it individually :hourglass:

Reply:
> Should we try v. 12.0.0-next.47 for react-dev-utils?
I think not good to use unreleased version (snapshots)

may be we can raise issue in their github and get it clarified or get info about next release.

Reply:
Added a direct dependency on `immer` to force bump it to `9.0.6` (force pushed)

Reply:
but will it break `react-dev-utils` ? we can test once, if it is fine. we can merge it.

Reply:
I smoke-tested the UI in a local build - it was fine, but I'm fairly new to the whole UI tools chain, so I'd appreciate if someone could double check.

Reply:
> but will it break react-dev-utils ?

`create-react-app` apparently uses this version combination on `main`.

https://github.com/facebook/create-react-app/blob/main/packages/react-dev-utils/package.json

Reply:
It turned out to be complicated... I did another force push to use `npm-force-resolutions` for bumping nested `immer` dependencies to `9.0.6`.